### PR TITLE
Farhan Ahmad Nurzi / Affiliate IB page not matching with the Design

### PR DESCRIPTION
**Pull request by farhan-nurzi-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/partners/affiliate-ib/_faq.tsx [hunk 1]
```diff
@@ -35,40 +41,50 @@ const RelativeContainer = styled(SectionContainer)`
...
 `
 
-const TopLeftDot = styled.img`
+const TopLeftDot = styled.img<DotProps>`
     width: 730px;
     position: absolute;
     top: 4px;
     left: 0;
+    right: 0;
+    margin: ${(props) => (props.is_row ? 'auto' : '0')};
```
Comment (NikitK-deriv) [inline src/pages/partners/affiliate-ib/_faq.tsx:50]: Let's use destructuring and proper logic here and below
 ```suggestion
    margin: ${({is_row}) => (is_row && 'auto')};
```

File: src/pages/partners/affiliate-ib/_faq.tsx [hunk 2]
```diff
@@ -35,40 +41,50 @@ const RelativeContainer = styled(SectionContainer)`
...
         width: 630px;
-        left: -50px;
+        margin: ${(props) => (props.is_row ? 'auto' : '0')};
+        left: ${(props) => (props.is_row ? '0' : '-50px')};
     }
 
     @media ${device.laptop} {
-        left: -250px;
+        margin: ${(props) => (props.is_row ? 'auto' : '0')};
+        left: ${(props) => (props.is_row ? '0' : '-250px')};
```
Comment (NikitK-deriv) [inline src/pages/partners/affiliate-ib/_faq.tsx:60]:
```suggestion
        left: ${({is_row}) => (!is_row && '-250px')};
```

File: src/pages/partners/affiliate-ib/_faq.tsx [hunk 3]
```diff
@@ -35,40 +41,50 @@ const RelativeContainer = styled(SectionContainer)`
...
     }
 
     @media ${device.tabletL} {
+        display: ${(props) => (props.is_row ? 'none' : 'block')};
```
Comment (NikitK-deriv) [inline src/pages/partners/affiliate-ib/_faq.tsx:64]:
```suggestion
        display: ${({is_row}) => (is_row ? 'none' : 'block')};
```

**Review by NikitK-deriv — COMMENTED** [review]
The rest LGMT 👍

File: src/pages/partners/affiliate-ib/_deriv-affiliate-programme.tsx [hunk 4]
```diff
@@ -208,7 +208,7 @@ const DerivAffiliateProgramme = () => {
                         )}
                     </SubtitleHeader>
                 </TitleWrapper>
-                <ComissionPlanContainer>
+                <ComissionPlanContainer background="transparent">
```
Comment (aswathy-deriv) [inline src/pages/partners/affiliate-ib/_deriv-affiliate-programme.tsx:211]: Add the style inside the component.Not the inline style

**Review by mohammad-hashemi-deriv — APPROVED** [review]

**Pull request closed without merging**